src/modalDescription.js

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 import './modal.css';
 
 function ModalDescription(props) {
-    // var table = props.scorecardData.tables[props.table];
-    // var row = props.scorecardData.tables[props.table].rows[props.row];
+    var isHowMuchTable = props.row.title === "Subtotal";
+    const howMuchDescription = "All candidates’ tweets over a period of three months are to be considered in the data. Each candidate will be ranked based on how high their score is. The candidate with the highest overall score will receive the full 30 points and each subsequent candidate will receive a weighted score based on how they compare to the highest and lowest ranking candidates. The candidate with the lowest overall score will receive zero points from this section. Keywords for Twitter advanced search include: “climate” “climate crisis” “climate change” “climate justice” “green new deal” “#greennewdeal” “global warming”. Twitter was chosen as the primary form of social media because all candidates have more Twitter followers than Facebook or Instagram followers. Our initial assessment will focus strictly on volume of tweets with room for more analysis."
```

Comment:
I didn't quite catch visually how this modal is different from the others & why this text is here rather than in the description in the data module - could you explain to me? (haha sorry, feel like i'm trying to play "spot the difference")

Reply:
Oh I see, there is only one row & it's subtotals ~ would it be weird to throw this in the data module?

Reply:
Sure! The 'how much' table will only have the 'subtotals' row so in the modal we'll need to change the row title so it doesn't just show "Subtotals" on the modal. For the description, I'm getting the sense from Alex that they're writing up a multi-paragraphed, more detailed description than the other categories, so it will make sense to just drop the html in the howMuchDescription variable so we can inject some html (like <p>, <a> tags etc) to make it look decent. Although you're right, we could just drop it in the data.js too. But I'm hoping to avoid embedding html(and then needing to escape the characters) in the JSON object